.pipelines/shared-cluster.yml

```diff
@@ -0,0 +1,88 @@
+# No PR triggers as it runs on a schedule
+pr: none
+trigger: none
+
+schedules:
+- cron: "0 1 * * Sun"
+  displayName: ⭕ Rebuild shared cluster Sun 1am (UTC)
+  always: true
+  branches:
+    include:
+    - master
+
+variables:
+  LOCATION: eastus
+  CLUSTER: aro-v4-shared
+
+jobs:
+- job: ⏬ Tear down shared cluster
+  pool:
+    vmImage: "Ubuntu-16.04"
+
+  container:
+    image: mcr.microsoft.com/azure-cli:latest
+
+  steps:
+  - template: ./templates/template-az-cli-login.yml
+    parameters:
+      workingDirectory: /
+      azureDevOpsJSONSPN: $(aro-v4-ci-devops-spn)
+  - template: ./templates/template-az-aro-extension-prod.yml
+  - displayName: 🧹 Delete cluster
+    script: |
+      az aro delete --yes --resource-group "$CLUSTER" --name "$CLUSTER"
+
+- job: ⏫ Set up shared cluster
+  pool:
+    vmImage: "Ubuntu-16.04"
```

Comment:
I think we should use existing Pools we have(?) @jim-minter ?

Reply:
Ubuntu-16.04 is the one pool that supports containers https://docs.microsoft.com/en-us/azure/devops/pipelines/process/container-phases?view=azure-devops#hosted-agents but I may be switching this over to not containers, although I think containers make this so much easier

Reply:
This is the poo we not control. We will be moving all jobs from this.